tests/buildtest.cmd

```diff
@@ -20,22 +29,19 @@ goto Usage
 if not defined __BuildArch set __BuildArch=x64
 if not defined __BuildType set __BuildType=debug
 if not defined __BuildOS set __BuildOS=Windows_NT
-if not defined __TestWorkingDir set "__TestWorkingDir=%__ProjectFilesDir%\..\binaries\tests\%__BuildOS%.%__BuildArch%.%__BuildType%"
-
-if not defined __LogsDir  set  "__LogsDir=%__ProjectFilesDir%..\binaries\Logs"
-if not defined __ProductSourceDir set "__ProductSourceDir=%__ProjectFilesDir%..\src"
 
+set "__TestBinDir=%__RootBinDir%\tests\%__BuildOS%.%__BuildArch%.%__BuildType%"
+set "__TestIntermediatesDir=%__RootBinDir%\tests\intermediates\%__BuildOS%.%__BuildArch%.%__BuildType%"
```

Comment:
We can have the same Intermediate Directory for all configurations as the current one looks like:
binaries\tests\intermediates\Common\Coreclr.TestWrapper\Windows_NT.x64.debug